### PR TITLE
Updated CHANGES.md for v1.12.0 release

### PR DESCRIPTION
**Pull request by tedsta**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,57 @@
+# Next Release
```
Comment (dlott) [inline CHANGES.md]: Should this be v1.12.0?  
And we add a new Changes.next.md that people update?

Comment (yuyantingzero) [inline CHANGES.md]: Release number

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,57 @@
...
+- Add Stencil2D benchmark (GH-1332)
+- Managed memcache service abstraction and a backend for AWS ElastiCache (GH-1235)
+- Add Apache Beam benchmark (GH-1324)
+- Add `image_project` to GCP YAML config options (GH-1319)
+- Add support for AWS i3
+- Optionally create failed run samples with --create_failed_run_samples (GH-1363)
+- Add --storage_size flag for mysql_service benchmark
+
+Enhancements:
+- Update Azure CLI to v2.0
```
Comment (yuyantingzero) [inline CHANGES.md]: PR number

File: CHANGES.md [hunk 3]
```diff
@@ -1,3 +1,57 @@
...
+- Add optional step parameter to IntegerList (GH-1330)
+- Upgrade to NVIDIA's latest version of CUDA Toolkit 8 (GH-1304)
+- Add ID attribute to GCE VMs (GH-1318)
+- Add ability to install netperf from local tar file (GH-1364)
+
+Bug fixes and maintenance updates:
+- Fix up README.md (GH-1350, GH-1354)
```
Comment (yuyantingzero) [inline CHANGES.md]: a bit more details?

File: CHANGES.md [hunk 4]
```diff
@@ -1,3 +1,57 @@
+# Next Release
+
+New features:
+- Azure container service (GH-1361)
+- VM Reboot
```
Comment (yuyantingzero) [inline CHANGES.md]: PR number, also maybe some more details.

File: CHANGES.md (hunk 2 above)
Comment (yuyantingzero) [inline CHANGES.md]: also mention it is a breaking change

File: CHANGES.md (hunk 1 above)
Comment (tedsta) [inline CHANGES.md]: The assumes all changes will be in the "# Next Release" section and automatically replaces it with the release number and creates a new empty "# Next Release" section.

Comment (tedsta) [inline CHANGES.md]: The assumes all changes will be in the "# Next Release" section and automatically replaces it with the release number and creates a new empty "# Next Release" section.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -23,6 +23,20 @@ use GitHub pull requests for this purpose.
 Contributions made by corporations are covered by a different agreement than
 the one above, the Software Grant and Corporate Contributor License Agreement.
 
+### Change log
+After creating a pull request, update the appropriate sections in CHANGES.next.md with a summary of your changes, then commit and push to the branch you are trying to merge.
```
Comment (yuyantingzero) [inline CONTRIBUTING.md]: seems good, but how are we going to inform contributors about this? during reviews?

Comment (dlott) [inline CONTRIBUTING.md]: yeah, with a link to these instructions.  people will learn quickly.

**Pull request merged**